en/developer/paywall.md

```diff
@@ -5,7 +5,7 @@
 > This is the technical part about the paywall. If you are looking for the user part, please check [that page instead](../user/articles/restricted.md).
 
 Read [this part of the documentation](../user/errors_during_fetching.md)
-to understand the configuration files, which are located under `vendor/j0k3r/graby-site-config/`. For most of the websites, this file
```

Comment:
No this can't be removed.
We precise ***where** files are located in the directory not in on the internet. Maybe you should better add a mention to the repo following the directy information

Reply:
I understand, but the 'vendor' directory doesn't exit anymore, does it? I can't find it.

Reply:
It's not in the wallabag repository but is created while installing wallabag.

Reply:
Got it, thanks